src/quo/components/text_combinations/view.cljs

```diff
@@ -29,7 +33,9 @@
            button-icon
            button-on-press
            customization-color
-           emoji-hash]
+           emoji-hash
+           emoji
+           emoji-background-color]
```

Comment:
@ibrkhalil - here the principle is similar to the other or component, we don't want to add override colors. In this I believe this is just `customization-color` which we pass as that for consistency among the quo components. You then have to resolve that accordingly